pom.xml

```diff
@@ -985,6 +975,39 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <property>
+                    <name>!withTrino</name>
+                </property>
+            </activation>
+            <modules>
+                <module>core</module>
+                <module>hive</module>
+                <module>flink</module>
+                <module>spark</module>
+                <module>ams</module>
+                <module>dist</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>include-trino</id>
+            <activation>
+                <property>
+                    <name>withTrino</name>
+                </property>
+            </activation>
+            <modules>
+                <module>core</module>
+                <module>hive</module>
+                <module>flink</module>
+                <module>spark</module>
+                <module>trino</module>
+                <module>ams</module>
+                <module>dist</module>
+            </modules>
+        </profile>
```

Comment:
This part seems a bit too complicated. Just use Profile to enable it, no need to use a property. "-Ptrino" is already enough.